core/profile.c

```diff
@@ -1297,7 +1306,9 @@ void create_plot_info_new(struct dive *dive, struct divecomputer *dc, struct plo
 {
 	int o2, he, o2max;
 #ifndef SUBSURFACE_MOBILE
-	init_decompression(dive);
+	struct deco_state plot_deco_state;
+	struct deco_state *ds = &plot_deco_state;
```

Comment:
Does one really need the additional `struct deco_state *ds` pointer here? One can pass &plot_deco_state later on.